apps/devportal/data/markdown/pages/learn/faq/xm-personalize/edge-serverless.md

```diff
@@ -1,29 +1,31 @@
 ---
 product: ['xm', 'personalize']
-title: 'Edge & Serverless hosting'
+title: 'Experience Edge & Serverless hosting'
 hasInPageNav: true
 hasSubPageNav: true
 cdpTags: ['xm', 'personalize']
 ---
 
-### Will changing my infrastructure to use serverless hosting (Edge+Vercel/Netlify, etc.) instead of CDs, secure that my application won&#39;t require further changes when moving into XM Cloud?
+### Will changing my infrastructure to use serverless hosting (Sitecore Experience Edge + Vercel/Netlify, etc.) instead of CDs, secure that my application won&#39;t require further changes when moving into XM Cloud?
 
 This will not necessarily secure the application by default. When moving to SaaS there are some components of the Sitecore XM solution that the customer will have to give up control over to move to SaaS. Some examples of XM capabilities that will not be available in XM Cloud are custom Solr indexes and use of Sitecore Forms.
 
 ### I already pay for Vercel/Netlify for other applications we have, can I use my current hosting with XP2XMP?
```

Comment:
There is an explanation of it on the main introduction page, but I agree this needs to be looked at. It makes it more difficult to understand the question and it doesn't help with keywords because nobody will google for that acronym. It only helps the author. I will try to do an FAQ-wide update to remove that terminology.